content/en/guides/get-started/commands.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Commands and Deployment
+description: Nuxt.js comes with a set of useful commands, both for development and production purpose.
+position: 4
+category: get-started
+csb_link: https://codesandbox.io/embed/github/nuxt-academy/guides-examples/tree/master/01_get_started/02_routing?fontsize=14&hidenavigation=1&theme=dark
+---
+
+Nuxt.js comes with a set of useful commands, both for development and production purpose.
+
+## Using in package.json
+
+You should put these commands in the `package.json`:
+
+```json
+"scripts": {
+  "dev": "nuxt",
+  "build": "nuxt build",
+  "start": "nuxt start",
+  "generate": "nuxt generate"
+}
+```
+
+Then, you can launch your commands via `yarn <command>` (example: `yarn dev`).
+
+## Development Environment
+
+To launch Nuxt in development mode with hot reloading:
+
+<code-group>
+  <code-block label="npx" active>
+
+```bash
+nuxt
+```
+
+  </code-block>
+
+  <code-block label="Yarn">
+
+```bash
+yarn dev
+```
+
+  </code-block>
+
+  <code-block label="NPM">
+
+```bash
+npm run dev
+```
+
+  </code-block>
+</code-group>
+
+## List of Commands
+
+You can run different commands depending on the [target](/guides/features/deployment-targets) :
+
+### target: `server` (default value)
+
+- **nuxt dev** - Launch a development server on localhost:3000 with hot-reloading.
+- **nuxt build** - Build your application with webpack and minify the JS & CSS (for production).
+- **nuxt start** - Start the production server (after running `nuxt build`). Use it for Node.js hosting like Heroku, Digital Ocean, etc.
+
+### target: `static`
+
+- **nuxt dev** - Launch a development server on localhost:3000 with hot-reloading.
+- **nuxt generate** - Build the application (if needed), generate every route as a HTML file and statically export to `/dist` (used for static hosting).
+- **nuxt start** - serve the dist/ directory like your static hosting would do (Netlify, Vercel, Surge, etc), great for testing before deploying.
+
+## Production Deployment
+
+Nuxt.js lets you choose between three modes to deploy your application: SSR, Static Generated, or SPA.
+
+### Server-Side Rendered Deployment (Universal SSR)
+
+To deploy a SSR application we use `target: server`, where server is the default value.
+
+<code-group>
+  <code-block label="npx" active>
+
+```bash
+nuxt build
+```
+
+  </code-block>
+  <code-block label="Yarn">
+
+```bash
+yarn build
+```
+
+  </code-block>
+  <code-block label="NPM">
+
+```bash
+npm run build
+```
+
+  </code-block>
+</code-group>
+
+Once you application is built you can use the `start` command to see a production version of your application
```

Comment:
~> to run your application in production mode

Maybe also adding a tip to use `PORT=4000 nuxt start` when using pwa module?

Reply:
For this, could the PWA module force the 4000 port with nuxt start @pi0 ?

Reply:
Or make the nuxt start port 4000 by default (when running it with target:static)?

Reply:
Both forcing methods are breaking changes since we don't know `nuxt start` is meant for serving in a real production environment or quickly locally testing. Using different port with static target makes sense since `nuxt start` is only meant for locally testing. That's why i suggested to mention this tip for static section (we can also do change in nuxt core)